From be37ed0c17773bb0508943b74fd3d8a92f68b9d5 Mon Sep 17 00:00:00 2001
From: ta264 <ta264@users.noreply.github.com>
Date: Mon, 11 Apr 2022 19:20:20 +0100
Subject: [PATCH] Fix duplicate definition of _RUNTIME_FUNCTION on linux-x86

---
 src/coreclr/inc/clrnt.h | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/src/coreclr/inc/clrnt.h b/src/coreclr/inc/clrnt.h
index a833da67bcc..b42ae8a7c9b 100644
--- a/src/coreclr/inc/clrnt.h
+++ b/src/coreclr/inc/clrnt.h
@@ -833,21 +833,6 @@ RtlVirtualUnwind_Unsafe(
 //
 
 #ifdef TARGET_X86
-#ifndef TARGET_UNIX
-//
-// x86 ABI does not define RUNTIME_FUNCTION. Define our own to allow unification between x86 and other platforms.
-//
-#ifdef HOST_X86
-typedef struct _RUNTIME_FUNCTION {
-    DWORD BeginAddress;
-    DWORD UnwindData;
-} RUNTIME_FUNCTION, *PRUNTIME_FUNCTION;
-
-typedef struct _DISPATCHER_CONTEXT {
-    _EXCEPTION_REGISTRATION_RECORD* RegistrationPointer;
-} DISPATCHER_CONTEXT, *PDISPATCHER_CONTEXT;
-#endif // HOST_X86
-#endif // !TARGET_UNIX
 
 #define RUNTIME_FUNCTION__BeginAddress(prf)             (prf)->BeginAddress
 #define RUNTIME_FUNCTION__SetBeginAddress(prf,addr)     ((prf)->BeginAddress = (addr))
-- 
2.25.1

